extensions/redis-client/deployment/src/main/java/io/quarkus/redis/client/deployment/DevServicesRedisProcessor.java

```diff
@@ -184,8 +184,11 @@ private RunningDevService startContainer(String connectionName, DevServicesConfi
         };
 
         return redisContainerLocator.locateContainer(devServicesConfig.serviceName, devServicesConfig.shared, launchMode)
-                .map(containerAddress -> new RunningDevService(Feature.REDIS_CLIENT.getName(), containerAddress.getId(),
-                        null, configPrefix + RedisConfig.HOSTS_CONFIG_NAME, containerAddress.getUrl()))
+                .map(containerAddress -> {
+                    String redisHost = REDIS_SCHEME + containerAddress.getHost() + ":" + containerAddress.getPort();
```

Comment:
```suggestion
                    String redisHost = REDIS_SCHEME + containerAddress.getUrl();
```
nitpick: We could use the `.getUrl()` method.